variables.tf

```diff
@@ -65,6 +65,12 @@ variable "description" {
   default     = null
 }
 
+variable "disable_execute_api_endpoint" {
+  type        = string
+  default     = false
+  description = "Whether clients can invoke the API by using the default execute-api endpoint. To require that clients use a custom domain name to invoke the API, disable the default endpoint"
```

Comment:
Please reorder these keys: description, type, default. Like on other variables already.

Reply:
Maybe you can use my fork instead https://github.com/davidkarlsen/terraform-aws-apigateway-v2/tree/mtls ? It has adressed these issues